src/datastation/scripts/fix_provenance_files.py

```diff
@@ -22,15 +22,21 @@
                      'xmlns:abr="http://www.den.nl/standaard/166/Archeologisch-Basisregister/" ' \
                      'xmlns:ddm="http://easy.dans.knaw.nl/schemas/md/ddm/"'
 
+provenance_schema_doc = etree.parse('http://easy.dans.knaw.nl/schemas/bag/metadata/prov/provenance.xsd')
+provenance_xmlschema = etree.XMLSchema(provenance_schema_doc)
 
-def validate(xml_path: str, xsd_path: str) -> bool:
-    xmlschema_doc = etree.parse(xsd_path)
-    xmlschema = etree.XMLSchema(xmlschema_doc)
+
+def validate(xml_path: str) -> bool:
     try:
         xml_doc = etree.parse(xml_path)
-        result = xmlschema.validate(xml_doc)
-        return result
-    except etree.XMLSyntaxError:
+        provenance_xmlschema.assertValid(xml_doc)
+        return True
+    except etree.XMLSyntaxError as e:
+        logging.error(e)
```

Comment:
We also expect syntax errors in many of the old provenance.xml's

Reply:
done